Modules/System/Azure Storage Services Authorization/src/StorServAuthImpl.Codeunit.al

```diff
@@ -50,6 +50,16 @@ codeunit 9063 "Stor. Serv. Auth. Impl."
         exit(StorServAuthSharedKey);
     end;
 
+    [NonDebuggable]
+    procedure ReadySAS(SASToken: Text)
```

Comment:
You're missing the return paramter ` Interface "Storage Service Authorization"`.

Reply:
Right, thanks, fixed.